A3A/addons/core/functions/Missions/fn_missionRequest.sqf

```diff
@@ -193,12 +193,9 @@ switch (_type) do {
                     [[_site],"A3A_fnc_LOG_Salvage"] remoteExec ["A3A_fnc_scheduler", 2];
                 };
                 case(_site in controlsX): {
-					private _roll = random 100;
-					if(_roll < 50) then {
-						[[_site],"A3A_fnc_LOG_Airdrop"] remoteExec ["A3A_fnc_scheduler",2];
-					} else {
-						[[_site],"A3A_fnc_LOG_Helicrash"] remoteExec ["A3A_fnc_scheduler", 2];
-					};
+					private _LOGmissions = ["A3A_fnc_LOG_Airdrop", 0.35, "A3A_fnc_LOG_Helicrash", 0.35, "A3A_fnc_LOG_Crashsite", 0.3];
+					private _LOGmission = selectRandomWeighted _LOGmissions;
+					[[_site],_LOGmission] remoteExec ["A3A_fnc_scheduler", 2];
```

Comment:
Use camel case for variable names, it's easier to read and follows standard conventions

```suggestion
					private _logMissions = ["A3A_fnc_LOG_Airdrop", 0.35, "A3A_fnc_LOG_Helicrash", 0.35, "A3A_fnc_LOG_Crashsite", 0.3];
					private _logMission = selectRandomWeighted _logMissions;
					[[_site],_logMission] remoteExec ["A3A_fnc_scheduler", 2];
```